content/fringes/pulppluginday.md

```diff
@@ -2,20 +2,36 @@
 title: "Pulp Plugin Day"
 date: 2018-11-23
 eventdate: "6th February 2019"
-draft: true
+draft: false
 ---
 Pulp is a platform for managing repositories of software packages, but it can store any kind of
```

Comment:
Want to unline-wrap this post?

Reply:
This sentence I think would work better with an 'and' in the middle (compound predicate). Maybe

Pulp is a platform for managing repositories of software packages and can store any kind of binary data (e.g. cat photos).